Test failures with python-dateutil 2.5.3

When installing `python-dateutil==2.5.0` or newer, the tests fail:

    ======================================================================
    FAIL: test_tzinfo_dateutil (icalendar.tests.test_timezoned.TestTimezoned)
    ----------------------------------------------------------------------
    Traceback (most recent call last):
      File "/tmp/guix-build-python-icalendar-3.9.2.drv-0/icalendar-3.9.2/src/icalendar/tests/test_timezoned.py", line 138, in test_tzinfo_dateutil
        self.assertTrue(date.tzinfo.__module__ == 'dateutil.tz')
    AssertionError: False is not true

    ----------------------------------------------------------------------
    Ran 84 tests in 0.165s

    FAILED (failures=1)

Using any older version seems to fix the problem.

cc @lfam

--- a/src/icalendar/tests/test_timezoned.py
+++ b/src/icalendar/tests/test_timezoned.py
@@ -135,8 +135,8 @@ class TestTimezoned(unittest.TestCase):
         # references: #73,7430b66862346fe3a6a100ab25e35a8711446717
         date = dateutil.parser.parse('2012-08-30T22:41:00Z')
         date2 = dateutil.parser.parse('2012-08-30T22:41:00 +02:00')
-        self.assertTrue(date.tzinfo.__module__ == 'dateutil.tz')
-        self.assertTrue(date2.tzinfo.__module__ == 'dateutil.tz')
+        self.assertTrue(date.tzinfo.__module__.startswith('dateutil.tz'))
+        self.assertTrue(date2.tzinfo.__module__.startswith('dateutil.tz'))
 
         # make sure, it's parsed properly and doesn't throw an error
         self.assertTrue(icalendar.vDDDTypes(date).to_ical()

